core/src/main/antlr4/org/apache/druid/math/expr/antlr/Expr.g4

```diff
@@ -1,5 +1,20 @@
 grammar Expr;
 
+// Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Is this better than putting the license header at the beginning of the file?

Reply:
@jihoonson putting it at the beginning would be better. I wasn't sure if the `grammar` keyword needed to be on the first line (like <?xml> needing to be on the first line of the file) and the example I saw had comments after `grammar` but I found another example with a proper header on line 1 (https://github.com/antlr/grammars-v4/blob/master/cpp/CPP14.g4). Will change, thanks!

Reply:
Ah thanks! I think they could come at the beginning in bash and python scripts as well. Would you please check them too?

Reply:
Ok, thanks.